src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1138 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Flowsheet Equilibrium Reactor Simulation and Optimization of Steam Methane Reforming\n",
+    "\n",
+    "\n",
+    "## Learning outcomes\n",
+    "\n",
+    "\n",
+    "- Call and implement the IDAES EquilibriumReactor unit model\n",
+    "- Construct a steady-state flowsheet using the IDAES unit model library\n",
+    "- Connecting unit models in a flowsheet using Arcs\n",
+    "- Fomulate and solve an optimization problem\n",
+    "    - Defining an objective function\n",
+    "    - Setting variable bounds\n",
+    "    - Adding additional constraints \n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "This example is adapted from S.Z. Abbas, V. Dupont, T. Mahmud, Kinetics study and modelling of steam methane reforming process over a NiO/Al2O3 catalyst in an adiabatic packed bed reactor. Int. J. Hydrogen Energy, 42 (2017), pp. 2889-2903\n",
+    "\n",
+    "Steam methane reforming (SMR) is one of the most common pathways for hydrogen production, taking advantage of chemical equilibria in natural gas systems. The process is typically done in two steps: methane reformation at a high temperature to partially oxidize methane, and water gas shift at a low temperature to complete the oxidation reaction:\n",
+    "\n",
+    "**C</sub>H<sub>4</sub> + H<sub>2</sub>O</sub> → C</sub>O</sub> + 3H<sub>2</sub>**  \n",
+    "**C</sub>O</sub> + H<sub>2</sub>O</sub> → C</sub>O<sub>2</sub> + H<sub>2</sub>**\n",
+    "\n",
+    "This reaction is often carried out in two separate reactors to allow for different reaction temperatures and pressures; in this example, we will minimize operating cost for a single reactor.\n",
+    "\n",
+    "The flowsheet that we will be using for this module is shown below with the stream conditions. We will be processing natural gas and steam feeds of fixed composition to produce hydrogen. As shown in the flowsheet, the process consists of a mixer M101 for the two inlet streams, a compressor to compress the feed to the reaction pressure, a heater H101 to heat the feed to the reaction temperature, and a EquilibriumReactor unit R101. We will use thermodynamic properties from the Peng-Robinson equation of state for this flowsheet.\n",
+    "\n",
+    "The state variables chosen for the property package are **total molar flows of each stream, temperature of each stream and pressure of each stream, and mole fractions of each component in each stream**. The components considered are: **CH4, H2O, CO, CO2, and H2** and the process occurs in vapor phase only. Therefore, every stream has 1 flow variable, 5 mole fraction variables, 1 temperature and 1 pressure variable. \n",
```

Comment:
It's minor, but could you use `<sub>`  tags to turn **H2** into **H<sub>2</sub>**, etc.?